docs_src/microservices/application/AppServiceConfigurable.md

```diff
@@ -53,66 +30,105 @@ the configuration in the `configuration.toml` file under the
+
+The following profiles and their purposes are provided with App Service Configurable. 
+
+- **blackbox-tests** - Profile used for black box testing the SDK 
+- **http-export** - Starter profile used for exporting data via HTTP. 
+  Requires further configuration which can easily be accomplished using environment variable overrides
+  - Required:
+    - `Writable_Pipeline_Functions_HTTPPostJSON_Parameters_url:[Your URL]`
+  - Optional: 
+    - `Writable_Pipeline_Functions_HTTPPostJSON_Parameters_persistOnError:["true"/"false"]`
+  - Optional: 
+    - `Writable_Pipeline_Functions_FilterByDeviceName_Parameters_DeviceNames:"[comma separated list]"`
+  - Optional: 
+    - `Writable_Pipeline_Functions_FilterByValueDescriptor_Parameters_ValueDescriptors: "[comma separated list]"`
+- **mqtt-export** - Starter profile used for exporting data via MQTT.
+  Requires further configuration which can easily be accomplished using environment variable overrides
+  - Required:
+    - `Writable_Pipeline_Functions_MQTTSend_Addressable_Address:[Your Address]`
+  - Optional: 
+    - `Writable_Pipeline_Functions_MQTTSend_Addressable`
+      - `_Port:["your port"]`
+      - `_Protocol:[tcp or tcps]`  
+      - `_Publisher:[your name]`
+      - `_User:[your username]`
+      - `_Password:[your passowrd`
+      - `_Topic:[your topic]`
+  - Optional: 
+    - `Writable_Pipeline_Functions_MQTTSend_Parameters`
+      - `_qos:["your quality or service"]`
+      - `_key:[your Key]`  
+      - `_cert:[your Certificate]`
+      - `_autoreconnect:["true" or "false"]`
+      - `_retain:["true" or "false"]`
+      - `_persistOnError:["true" or "false"]`
+- **rules-engine** - Profile used to push Event messages to the Rules Engine via **ZMQ** Message Bus.
+- **rules-engine-mqtt** - Profile used to push Event messages to the Rules Engine via **MQTT** Message Bus.
+- **rules-engine-redis** Profile used to push Event messages to the Rules Engine via **RedisStreams** Message Bus.
+- **sample** - Sample profile with all available functions declared and a sample pipeline. Provided as a sample that can be copied and modified to create new custom profiles.
+
+> *Note: Functions can be declared in a profile but not used in the pipeline `ExecutionOrder` allowing them to be added to the pipeline `ExecutionOrder` later at runtime if needed. *
+
+If you need to run multiple instances with the same profile, e.g. `http-export`, but configured differently, you will need to override the service key with a custom name for one or more of the services. This is done with the `-sk/-serviceKey` command-line option or the `EDGEX_SERVICE_KEY` environment variable. See the `Command-line Options` and `Environment Overrides` documentation located in the [Application Functions SDK](./ApplicationFunctionsSDK.md#command-line-options) section for more detail.
 
 ## Input Data Not An EdgeX Event
```

Comment:
This section is showing as italicized which indicates to me that its not quite formatted correctly.

Reply:
Not sure why... Now it has been moved up as a !! note.